h4l/calibration/jets.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+
+"""
+Jet energy calibration methods.
+"""
+
+from columnflow.calibration import Calibrator, calibrator
+from columnflow.calibration.cms.jets import jec, jer
+from columnflow.util import maybe_import
+
+ak = maybe_import("awkward")
+
+
+# custom jec calibrator that only runs nominal correction
+jec_nominal = jec.derive("jec_nominal", cls_dict={"uncertainty_sources": []})
```

Comment:
Ah right, it seems the `jec_nominal` is now in the upstream columnflow repo as well. I had copied the code from another analysis which was probably written before this was available.

We can remove it in principle, but maybe it would be worth to keep this here as an example on how to derive custom calibrators. That could actually be useful for implementing JEC uncertainties like you suggest.